src/App.js

```diff
@@ -18,6 +25,22 @@ function App() {
       <Logo />
       <div className="App">
         {/* STEP 4 - Render your components here and be sure to properly import/export all files */}
+        <div className="screen">
+          <Display number={1024}/>
+        </div>
+            <div className="Buttons">
+              <div className="specificButtons">
+                <div className="special">
+                    < Specials/>
+                  </div>
+                <div className="numbers">
+                  <Numbers/>
+                </div>
+                <div className="operators">
+                  <Operators/>
+                </div>
+            </div>
+        </div>  
       </div>
     </div>
   );
```

Comment:
I like how you added more divs to make styling easier and flexing that flex-box skill!